packages/utils/src/instrument/fetch.ts

```diff
@@ -116,7 +116,7 @@ function instrumentFetch(onFetchResolved?: (response: Response) => void, skipNat
 }
 
 async function resolveResponse(res: Response | undefined, onFinishedResolving: () => void): Promise<void> {
-  if (res && res.body) {
+  if (res && res.body && typeof res.body.getReader === 'function') {
```

Comment:
l: Maybe it is enough to do `&& res.body.getReader`? Saves a tiny bit of bytes...